### DISCUSSION TITLE
Can someone give an example of working placeholder code

### DISCUSSION THREAD

**Discussion by paridhikhaitan** [ueberdosis/tiptap#805] [General Questions]
I've followed the placeholder setup from TipTap and have tried everything to get it to work. I can see that on the DOM, the "is-editor-empty" class is rendered. However, my placeholder is not showing up. Does anybody have any ideas why this could be?

**Comment by hanspagel** [comment]
Did you copy the styling?

https://github.com/ueberdosis/tiptap/blob/510a135c6d454bcf1e583f19843476f676583525/examples/Components/Routes/Placeholder/index.vue#L44-L51
